distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -13,6 +13,7 @@
  */
```

Comment:
My code formatter was to "reorganize", so the file diff looks chopped-up. Something about my PR changes the runtime environment enough so that these tests fail, unless you set a breakpoint somewhere. In debug mode, the tests pass. A ticket was created for this issue by @vinamartin (DDF-2670).